Readme.md

```diff
@@ -0,0 +1,51 @@
+# BeCJK & History
+
+![screenshot](screenshot.png)
+
+BeCJK, is an input method for the legancy BeOS, first Developed by
+Anthony.Lee.
+
+Now, this copy of code is gotten from Andraw Bachmann.
+
+As time goes on, expecially with Haiku PM, BeCJK can't work normally on
+recently release of Haiku.
+
+# Building
+
+## Pre-requisite software
+
+  - mercurial, get current code of BeCJK
```

Comment:
shouldn't need mercurial now that it is hosted on github.  You'd need git instead.

Reply:
Okay.